lib/site_prism/waiter.rb

```diff
@@ -18,5 +18,11 @@ def self.default_wait_time
       warn 'default_wait_time is now deprecated. This will be removed in an upcoming release.'
       Capybara.default_max_wait_time
     end
+
+    def self.wait_with_capybara(timeout: nil, or_raise: nil)
+      timeout = Capybara.default_max_wait_time if timeout.nil?
+      raise or_raise unless (result = Capybara.using_wait_time(timeout) { yield }) || !or_raise
+      result
+    end
```

Comment:
TBH, I'm not really sure that this deserves to be off in it's own class like this, but that's where the other wait stuff was living.  In the future we might consider folding it into a simple mixin instead.

Reply:
If it's going to be wiped out down the line we'll worry about it later. I'm happy with this methodology